docs/installation-and-operations/system-requirements/README.md

```diff
@@ -12,10 +12,51 @@ provide any official support for them.
 
 ## Server
 
-### Hardware
+The server hardware requirements should be rougly the same for both the packaged installation and docker (both all-in-one container and compose).
 
+### Minimum Hardware Requirements
+
+* __CPU:__ Quad Core CPU (>= 2ghz)
 * __Memory:__ 4096 MB
-* __Free disk space:__ 4 GB
+* __Free disk space:__ 20 GB
+
+This is for a single Server running OpenProject for up to 200 total active users. Depending on your number of peak users, these requirements might vary drastically.
```

Comment:
```suggestion
This is for a single server running OpenProject for up to 200 total users. Depending on your number of concurrent users,  these requirements might vary drastically.
```

I reworded this sentence since I find `peak` not to be really clear. After some googling, performance statistics often seem to mention either _concurrent users_ (performing any request on the page within the timeframe) and _simultaneous users_ (number of users performing the same request at the same time). I'd say we stick with `concurrent users` then.